CHANGES.txt

```diff
@@ -7,6 +7,7 @@ V3.0.19
  - Creating in the config new constants to prevent sqlite locks when updating any set
     - SCIPION_UPDATE_SET_ATTEMPTS: Number of attempts when trying a set update. The default value is 3
     - SCIPION_UPDATE_SET_ATTEMPT_WAIT: Time in seconds to wait between updates. The default value is 2 seconds
+ - In _showWizard, passing the paramName as a form variable so the wizard can check from which parameter it was called
```

Comment:
@DaniDelHoyo , please add a "Developers" section above since this is more a developers related functionality